analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -99,13 +103,6 @@ starfusionDF <- read_tsv(file.path(dataDir, "fusion-starfusion.tsv.gz"))
 # read in DGD data 
 dgd_fusion <- readr::read_tsv(file.path(dataDir, "fusion-dgd.tsv.gz"))
```

Comment:
For lines 94-102 here, I think we need a new approach. What this is doing is that it is taking any sample which had a fusion call in any gene and adding it to the fusion dataframes. BUT, what I think we want is _any sample assayed for fusions_ in these dataframes. Any which had RNA-Seq or DGD fusion panel. Can you update this? It is needed downstream for the HGG subtyping (possibly LGG).

Reply:
Note: I updated this